src/emulator/modules/SceLibKernel/src/SceLibKernel.cpp

```diff
@@ -744,11 +748,11 @@ EXPORT(int, sceKernelCreateRWLock) {
 EXPORT(SceUID, sceKernelCreateSema, const char *name, SceUInt attr, int initVal, int maxVal, SceKernelSemaOptParam *option) {
     assert(name != nullptr);
     assert(attr == 0);
-    assert(initVal == 0);
```

Comment:
Thanks. Is the minimum bound still 0? If so, might be worth adding a >= check.

Reply:
Can't really say if the minimum bound is 0 but i think so.